docs/develop/convert-javascript-to-typescript.md

```diff
@@ -43,18 +32,52 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 ## Convert the add-in project to TypeScript
 
-1. Find the **Home.js** file and rename it to **Home.ts**.
+### Add Nuget packages
 
-1. Find the **./Functions/FunctionFile.js** file and rename it to **FunctionFile.ts**.
+1. Open the Nuget package manager by choosing **Tools > Nuget Package Manager > Manage Nuget Packages for Solution**
+1. Select the **Browse** tab and search for **Microsoft.TypeScript.MSBuild**. Install this package to the ASP.NET web project, or update it if it's already installed. The ASP.NET web project has your project name with the text `Web` appended to the end. This will ensure the project will transpile to JavaScript when the build runs.
```

Comment:
Thanks for making the change. I think this is clearer, but is there any disadvantage to installing the package to the whole solution, instead of one specific project?

Reply:
I don't think so, but it just seems weird because the "manifest" project just sideloads your add-in. It doesn't use TypeScript. I don't want to give the impression that it has some dependency on TypeScript.